DataTanker/AccessMethods/BPlusTree/BPlusTree.cs

```diff
@@ -12,7 +12,7 @@
     /// </summary>
     /// <typeparam name="TKey">The type of keys</typeparam>
     /// <typeparam name="TValue">The type of values</typeparam>
-    internal class BPlusTree<TKey, TValue> : IBPlusTree<TKey, TValue> 
+    public class BPlusTree<TKey, TValue> : IBPlusTree<TKey, TValue> 
```

Comment:
What is the reason for increasing accessibility of internals?

Reply:
was a mistake. Meanwhile i completly forgot this was still open